docs/HowToFAQ/FAQRedundantCase.md

```diff
@@ -0,0 +1,33 @@
+---
+title: I just realized that a function I was compiling had a type-error inside a match case.  Instead of giving a compile error I was getting a redundant clause warning for the second case. What is the reason for this?
+---
+
+## Question
+
+I just realized that a function I was compiling had a type-error inside a match case.  Instead of giving a compile error I was getting a redundant clause warning for the second case. What is the reason for this?
```

Comment:
```suggestion
I just realized that a function I was compiling had a type error inside a match case.  Instead of giving a compile error I was getting a redundant clause warning for the second case. What is the reason for this?
```